client/src/components/chess-square/ChessSquare.test.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import ChessSquare from './ChessSquare';
+
+describe('<ChessSquare />', () => {
+  const fn = () => true;
+
+  test('should render empty dark unhighlighted chess square', () => {
+    const wrapper = shallow(<ChessSquare isDark gridPosition="a7" highlight={false} handleSelect={fn} />);
+    expect(wrapper.prop('className')).toBe('dark-square');
+    expect(wrapper.prop('grid-position')).toBe('a7');
+  });
+
+  test('should render empty light unhighlighted chess square', () => {
+    const wrapper = shallow(<ChessSquare isDark={false} gridPosition="a7" highlight={false} handleSelect={fn} />);
+    expect(wrapper.prop('className')).toBe('light-square');
+  });
+
+  test('should render chess square with piece', () => {
+    const wrapper = shallow(<ChessSquare isDark pieceType="b" pieceColor="b" gridPosition="c8" highlight={false} />);
```

Comment:
there is no peice type 'b', how is it rendering?

Reply:
Piece type 'b' refers to Bishop.